book/overlays.md

```diff
@@ -186,13 +186,31 @@ Sometimes, you might want to remove an overlay, but keep all the custom definiti
 
 (spam)> def eggs [] { "eggs" }
 
-(spam)> overlay remove --keep spam
+(spam)> overlay remove --keep-custom spam
 
 (zero)> eggs
 eggs
 ```
 
-The `--keep` flag does exactly that.
+The `--keep-custom` flag does exactly that.
+
+One can also keep a set of environment variables that were defined inside an overlay, but remove the rest:
```

Comment:
I'd rephrase it to something like "One can also keep a list of environment variables that were defined inside an overlay, but remove the rest, using the `--keep-env` flag:" (note the set->list change as well)

Reply:
yep, makes sense :ok_hand:

Reply:
should be addressed in [320ac57](https://github.com/nushell/nushell.github.io/pull/565/commits/320ac576c09e527acaae2a8caac9c2cc7ad4e72c) :yum:

Reply:
I meant the `--keep-env` flag should still be mentioned in the text. It just felt like the "
The `--keep-env` flag does exactly that." line later on was a bit out of place.

Reply:
ooooh sorry, i was too much focused on the `set->list` change :eyes:
got it now :wink:

Reply:
there should be the changes :relieved:
[1a2a41](https://github.com/nushell/nushell.github.io/pull/565/commits/1a2a41c669d34f67d86e75adbcc7a4f069ecfdfd)